src/nvim/grid.c

```diff
@@ -1078,6 +1078,112 @@ void grid_del_lines(ScreenGrid *grid, int row, int line_count, int end, int col,
   }
 }
 
+static void grid_draw_bordertext(VirtText vt, int col, int winbl, const int *hl_attr,
```

Comment:
@luukvbaal @bfredl OK with moving these functions from `src/nvim/drawscreen.c` to `grid.c` ?